specs/pvs/corla.pvs

```diff
@@ -203,9 +209,13 @@ END information_systems
 %%% requirements. Preferably redundant servers would be hosted in
 %%% separate facilities, with separate and independent power systems
 %%% and networks.
-%%%
```

Comment:
Are you intending to merge two paragraphs here? (that's what's happening with the LaTeX document when you delete this line)